spec/rubocop/cop/rspec/pending_without_reason_spec.rb

```diff
@@ -1,295 +1,689 @@
 # frozen_string_literal: true
 
 RSpec.describe RuboCop::Cop::RSpec::PendingWithoutReason, :ruby27 do
-  context 'when pending/skip has a reason inside an example' do
-    it 'registers no offense' do
-      expect_no_offenses(<<~'RUBY')
-        RSpec.describe Foo do
-          it 'does something' do
-            pending 'reason'
-            pending "#{reason}"
-            pending `echo #{reason}`
-            skip 'reason'
-            skip "#{reason}"
-            skip `echo #{reason}`
-          end
-        end
-      RUBY
+  context 'when Strict: true' do
+    let(:cop_config) { { 'Strict' => true } }
+
+    context 'when pending/skip has a reason inside an example' do
+      it 'registers no offense' do
+        expect_no_offenses(<<~'RUBY')
+          RSpec.describe Foo do
+            it 'does something' do
+              pending 'reason'
+              pending "#{reason}"
+              pending `echo #{reason}`
+              skip 'reason'
+              skip "#{reason}"
+              skip `echo #{reason}`
+            end
+          end
+        RUBY
+      end
```

Comment:
If this is common between the two styles, does it make sense to use a shared example?

It's kind of hard to understand the difference between the two.

Reply:
The behaviorally different specs are moved to the top of each context, and the good/bad cases have also been reviewed. How does that look?